src/styles/_catalog.scss

```diff
@@ -14,9 +13,15 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition: 300ms;
```

Comment:
`transition` is a shorthand property, use `transition-duration` if you only style the duration.